cn_indian_payroll/cn_indian_payroll/overrides/salary_slip.py

```
@@ -559,12 +523,255 @@ def add_reimbursement_taxable_new_doc(self):
 
 
 
+    def update_declaration_with_auto_calculation(self):
```

Ruff (0.8.2)

`update_declaration_with_auto_calculation` is too complex (54 > 10)

(C901)